sdk/program/src/stake/instruction.rs

```diff
@@ -261,6 +267,27 @@ pub enum StakeInstruction {
     ///   2. `[]` Reference vote account that has voted at least once in the last
     ///      `MINIMUM_DELINQUENT_EPOCHS_FOR_DEACTIVATION` epochs
     DeactivateDelinquent,
+
+    /// Redelegate activated stake to another vote account.
+    ///
+    /// Upon success:
+    ///   * the balance of the delegated stake account will be reduced to the rent exempt minimum,
+    ///     and scheduled for deactivation.
```

Comment:
I wasn't sure about additional SOL in the account -- Is this more correct?
```suggestion
    ///   * the balance of the delegated stake account will be reduced to the undelegated amount in the account (rent exempt minimum and any additional SOL not part of the delegation),
    ///     and scheduled for deactivation.
```

Reply:
yep, that's better. I'll make that change